core/kazoo_documents-1.0.0/src/kzd_app.erl

```diff
@@ -0,0 +1,195 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2015, 2600Hz
+%%% @doc
+%%% Account document
+%%% @end
+%%% @contributors
+%%%   Peter Defebvre
+%%%-------------------------------------------------------------------
+-module(kzd_app).
+
+-export([fetch/2]).
+
+-export([
+    id/1
+    ,is_published/1
+    ,name/1
+    ,i18n/1
+    ,tags/1
+    ,icon/1
+    ,api_url/1
+    ,source_url/1
+    ,author/1
+    ,version/1
+    ,license/1
+    ,price/1
+    ,screenshots/1
+    ,urls/1
+    ,account_id/1
+]).
+
+-include("kz_documents.hrl").
+
+-define(PUBLISHED, <<"published">>).
+-define(NAME, <<"name">>).
+-define(I18N, <<"i18n">>).
+-define(TAGS, <<"tags">>).
+-define(ICON, <<"icon">>).
+-define(API_URL, <<"api_url">>).
+-define(SOURCE_URL, <<"source_url">>).
+-define(AUTHOR, <<"author">>).
+-define(VERSION, <<"version">>).
+-define(LICENSE, <<"license">>).
+-define(PRICE, <<"price">>).
+-define(SCREENSHOTS, <<"screenshots">>).
+-define(URLS, <<"urls">>).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec fetch(api_binary(), api_binary()) -> {'ok', wh_json:object()} | {'error', any()}.
+fetch('undefined', _) ->
+    {'error', 'account_id_undefined'};
+fetch(_, 'undefined') ->
+    {'error', 'app_id_undefined'};
+fetch(Account, Id) ->
+    AccoundDb = wh_util:format_account_id(Account, 'encoded'),
+    couch_mgr:open_cache_doc(AccoundDb, Id).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec id(wh_json:object()) -> ne_binary().
+id(JObj) ->
+    wh_doc:id(JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec is_published(wh_json:object()) -> ne_binary().
+is_published(JObj) ->
+    wh_json:is_true(?PUBLISHED, JObj, 'true').
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec name(wh_json:object()) -> ne_binary().
+name(JObj) ->
+    wh_json:get_ne_binary_value(?NAME, JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec i18n(wh_json:object()) -> ne_binary().
```

Comment:
Can you also fix the spec in kzd_apps_store: "kzd_apps_store.erl:97: Invalid type specification for function kzd_apps_store:set_blacklist/2"